hie-compat/hie-compat.cabal

```diff
@@ -1,41 +1,59 @@
-cabal-version:       1.22
-name:                hie-compat
-version:             0.3.1.2
-synopsis:            HIE files for GHC 8.8 and other HIE file backports
-license:             Apache-2.0
+cabal-version: 1.22
+name: hie-compat
+version: 0.3.1.2
+synopsis: HIE files for GHC 8.8 and other HIE file backports
+license: Apache-2.0
 description:
   Backports for HIE files to GHC 8.8, along with a few other backports
   of HIE file related fixes for ghcide.
-
```

Comment:
This line was deleted because blank lines in the `description` field must be replaced by a lone period (`foo\n.\nbar`). You can see on Hackage that the description is all one paragraph: https://hackage.haskell.org/package/hie-compat-0.3.1.2

If you don't want to do that, you could upgrade to `cabal-version: 3.0`. You can see where Gild handles this here: https://github.com/tfausak/cabal-gild/blob/51425c05c67e6c335ba9024ced779669e3a4aca0/source/library/CabalGild/Unstable/Action/ReflowText.hs#L34

Here's the upstream Cabal PR for changing this behavior: https://github.com/haskell/cabal/pull/5981